tests/integration/release/test_runner.py

```diff
@@ -16,13 +16,31 @@
         (["--dataset", "wic", "--year", "2015"]),
         # (["--dataset", "wic", "--population", "USA", "--state", "RI", "--year", "2015"]),
     ],
+    ids=["1", "2", "3", "4"],
 )
-def test_release_tests(pytest_args: list[str]) -> None:
+def test_release_tests(
+    pytest_args: list[str], release_output_dir: Path, request: pytest.FixtureRequest
+) -> None:
     os.chdir(Path(__file__).parent)  # need this to access cli options from conftest.py
-    base_cmd = ["pytest", "--release", "test_release.py", "--check-max-tb=1000"]
+    base_cmd = [
+        "pytest",
+        "--release",
+        "test_release.py",
+        "--check-max-tb=1000",
+        "--population",
+        "USA",
+        f"--output-dir={release_output_dir}",
+    ]
     cmd = base_cmd + pytest_args
-    result = subprocess.run(cmd, capture_output=True, text=True)
-    assert result.returncode == 0
+
+    # log using job id
+    job_id = request.node.callspec.id
```

Comment:
I think this is even more of a reason to descriptively handle job IDs. `pytest_wic_2015.o` is a lot more informative than `pytest_4.o`.